src/jquantstats/_stats.py

```
@@ -8,7 +8,7 @@
 from scipy.stats import norm
 
 
-@dataclasses.dataclass(frozen=True)
+@dataclasses.dataclass(frozen=True)  # noqa: PLR0904
```

Ruff (0.14.2)

Unused `noqa` directive (non-enabled: `PLR0904`)

Remove unused `noqa` directive

(RUF100)